tests/Php72/Php72Test.php

```diff
@@ -48,4 +48,18 @@ public function testPhpOsFamily()
           $this->assertTrue(defined('PHP_OS_FAMILY'));
           $this->assertSame(PHP_OS_FAMILY, p::php_os_family());
     }
+
+    /**
+     * @covers Symfony\Polyfill\Php72\Php72::spl_object_id
+     */
+    public function testSplObjectId()
+    {
+        $obj = new \stdClass();
+        $id = spl_object_id($obj);
```

Comment:
please test that calling `spl_object_id` for non-object works the same as native function